vlib/v/parser/parser.v

```
@@ -3690,7 +3690,7 @@ fn (mut p Parser) const_decl() ast.ConstDecl {
 		name := p.check_name()
 		end_comments << p.eat_comments()
 		if util.contains_capital(name) {
```

```suggestion
		if !p.is_translated && util.contains_capital(name) {
```

---

The doom code has `[translated]` on its `module xyz` statements, and `!p.is_translated` will silence that error for such code.

---

Ok, Thanks !!!